llvm/test/Transforms/LoopVectorize/intrinsic.ll

```diff
@@ -1058,18 +1101,19 @@ for.end:                                          ; preds = %for.body, %entry
   ret void
 }
 
+define void @fabs_libm(ptr nocapture %x) nounwind {
 ; CHECK: fabs_libm
-; CHECK:  call <4 x float> @llvm.fabs.v4f32
+; CHECK: llvm.fabs.f32
```

Comment:
This is a change in the test case? Are the attributes needed here?

Reply:
The attributes were indeed necessary. Thanks for catching! Fixed now.